pkg/datapath/linux/probes/probes.go

```diff
@@ -205,6 +205,76 @@ func (p *ProbeManager) SystemConfigProbes() error {
 	return nil
 }
 
+// GetRequiredConfig performs a check of mandatory kernel configuration options. It
+// returns a map indicating which required kernel paramaters are enabled - and which are not
+// GetRequiredConfig is being used by CLI "cilium kernel-check"
+func (p *ProbeManager) GetRequiredConfig() (supported map[KernelParam]bool) {
+	config := p.features.SystemConfig
+	kernelParams := make(map[KernelParam]bool)
+
+	if !config.ConfigBpf.Enabled() {
+		kernelParams["CONFIG_BPF"] = false
+	} else {
+		kernelParams["CONFIG_BPF"] = config.ConfigBpf.Enabled()
+	}
```

Comment:
I meant:
```suggestion
	kernelParams["CONFIG_BPF"] = config.ConfigBpf.Enabled()
```
And similarly for the other blocks, no?

Reply:
missed that. fixed.